draft-ietf-taps-impl.md

```diff
@@ -327,31 +327,29 @@ Branching for derived endpoints is the final step, and may have multiple layers
 
 Implementations should sort the branches of the tree of connection options in order of their preference rank. 
 Leaf nodes on branches with higher rankings represent connection attempts that will be raced first.
-Implementations should order the branches to reflect the preferences expressed by the application for its new connection, including Protocol and Path Selection Properties, which are specified in {{I-D.ietf-taps-interface}}. 
+Implementations should order the branches to reflect the preferences expressed by the application for its new connection, including Selection and Connection Properties, which are specified in {{I-D.ietf-taps-interface}}. 
 
 In addition to the properties provided by the application, an implementation may include additional criteria such as cached performance estimates, see {{performance-caches}}, or system policy, see {{role-of-system-policy}}, in the ranking.
-Two examples of how the Protocol and Path Selection Properties may be used to sort branches are provided below:
+Two examples of how the Selection and Connection Properties may be used to sort branches are provided below:
 
-* Interface Type:
+* Selection Property "Interface Instance or Type":
 If the application specifies an interface type to be preferred or avoided, implementations should rank paths accordingly.
 If the application specifies an interface type to be required or prohibited, we expect an implementation to not include the non-conforming paths into the three.
 
-* Capacity Profile:
+* Connection Property "Capacity Profile":
 An implementation may use the Capacity Profile to prefer paths optimized for the application's expected traffic pattern according to cached performance estimates, see {{performance-caches}}:
```

Comment:
I think there is a matching Selection Property missing in the API description, if it should be used for selection.

Reply:
I think this would duplicate this property a second time and leads to confusion. I really prefer using the early-provided connection property here.

Reply:
@philsbln this isn't really a discussion for this PR (which is just trying to reflect the API in the implementation draft).
Anyway, to answer your concern: the capacity profile now exists in the interface draft as a Connection Property and as a per-Message override; the latter contains a statement about being inconsistent, and being fixed in the future.

Let's not get hung up on this now in this PR and fix the capacity profile later.